web/src/games/secretDracula/oldBoard.tsx

```diff
@@ -0,0 +1,778 @@
+import * as React from 'react';
+import { IGameArgs } from 'gamesShared/definitions/game';
+import { GameLayout } from 'gamesShared/components/fbg/GameLayout';
+import { Ctx } from 'boardgame.io';
+
+import { IG } from './interfaces';
+
+import { BVampirePolicies } from './components/bvampirepolicy';
+import { BPlayer } from './components/bplayer';
+import { BHumanPolicies } from './components/bhumanpolicy';
+import { BElectionTracker } from './components/belectiontracker';
+
+import { BVote } from './phases/vote/bvote';
+import { BEndVote } from './phases/vote/bvoteresults';
+import { BDiscard } from './phases/discardVeto/bdiscard';
+import { BPeek } from './phases/special/bpeek';
+import { BShowPlayer } from './components/bshowplayer';
+
+import css from './board.css';
+
+interface IBoardProps {
+  G: IG;
+  ctx: Ctx;
+  moves: any;
+  playerID: string;
+  gameArgs?: IGameArgs;
+}
+
+export class Board extends React.Component<IBoardProps> {
```

Comment:
It doesnt seem like this file is being used. Dead code makes it so much harder for people to understand the codebase. We will always have the previous board on git history.

Please delete this file.